### PR TITLE
Homework 7

### PR DESCRIPTION
**Pull request by Olga1A**

### CODE REVIEW

File: src/test/java/org/example/CalculatorTest.java [hunk 1]
```diff
@@ -1,49 +1,88 @@
...
-        assertFalse(myCalculator.isGreater(8, 8));
+        Calculator myCalculatorInst = new Calculator();
+        assertFalse(myCalculatorInst.isGreater(8, 8));
     }
 
     @Test
     public void secondValueIsGreater() {
-        Calculator myCalculator = new Calculator();
+        Calculator myCalculatorInst = new Calculator();
         assertFalse(8 < 7);
```
Comment (vladimir-tl) [inline src/test/java/org/example/CalculatorTest.java]: myCalculatorInst.isGreater(8, 7));

Comment (vladimir-tl) [reply src/test/java/org/example/CalculatorTest.java]: please fix it

Comment (Olga1A) [reply src/test/java/org/example/CalculatorTest.java]: Line 46 code amended

Comment (vladimir-tl) [reply src/test/java/org/example/CalculatorTest.java]: ok

**Review by vladimir-tl — APPROVED** [review]

**Pull request merged**